it/apps/src/main/content/jcr_root/apps/forms-core-components-it/customLocaleAddition/LocaleClientlibs/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    cssProcessor="[default:none,min:none]"
+    jsProcessor="[default:none,min:none]"
+    categories="[core.forms.components.runtime.all]"
```

Comment:
Please fix the categories. you can name it in your choice, but add a dependency on runtime.base.